README.md

```diff
@@ -287,6 +296,466 @@ pnpm build
+- In the CRM Admin Panel, go to **Channels > Unipile**
+- Enter the **API Key** and any other required information
+- Click **Save Credentials**
+- Click **Run Health Check** to verify the connection
+
+---
+
+## Environment Configuration
+
+Environment configuration is managed through `.env` files for each service. These files contain critical settings for database connections, API endpoints, authentication, and service-specific configurations.
+
+### Core Environment Variables
+
+#### Main .env File (Backend Configuration)
+The main `.env` file contains backend configurations, secrets, and deployment settings.
+
+#### Key Variables:
+- Configure database connections, API endpoints, and service-specific settings for each service
+- Ensure proper authentication tokens and credentials are set for inter-service communication
+
+### Service-Specific Environment Configurations
+
+#### DAL (Data Access Layer) Environment Variables
+```
+# Environment Configuration
+NODE_ENV="development" # Options: 'development', 'production', 'test'
+DAL_PORT="6001"            # The port your server will listen on
+DAL_HOST="localhost"       # Hostname for the server
+
+DAL_CORS_ORIGIN="http://localhost:3000" # Allowed CORS origin, adjust as necessary
+
+DAL_COMMON_RATE_LIMIT_MAX_REQUESTS="100" # Max number of requests per window per IP
+DAL_MINUTE_TO_LAUNCH="5"
+DAL_STRAPI_API_URL="http://localhost:1337/api/"
+DAL_STRAPI_API_TOKEN=""
+DAL_CHECK_TIME="1440" # This variable helps journeys to understand when to close processed journey job and open new 1 for checking default is 1 day
+DAL_JOB_FAIL_LIFE_TIME_DAYS="1"
+DAL_JOB_COMPLETED_LIFE_TIME_DAYS="1"
+
+DAL_REDIS_PORT="6379"
+DAL_REDIS_HOST="localhost"
+
+DAL_BASIC_AUTH_USERNAME=""
+DAL_BASIC_AUTH_PASSWORD=""
+
+DAL_WORKER_COUNT="4"
+DAL_JOB_CONCURRENCY="1"
+
+DAL_DATABASE_CLIENT="postgresql"
+DAL_DATABASE_HOST="localhost"
+DAL_DATABASE_PORT=5433
+DAL_DATABASE_NAME=""
+DAL_DATABASE_USERNAME=""
+DAL_DATABASE_PASSWORD=""
+DAL_DATABASE_RDS=false
+DAL_DATABASE_SSL_SELF=false
+
+DAL_SMTP_HOST="localhost"
+DAL_SMTP_PORT="587"
+DAL_SMTP_USER="username"
+DAL_SMTP_PASS="password"
+DAL_SMTP_FROM=""
+
+# Basic Auth (optional)
+DAL_BASIC_AUTH_USERNAME="admin"
+DAL_BASIC_AUTH_PASSWORD="admin"
+
+# Shared URLs
+STRAPI_URL="http://localhost:1337/api/"
+COMPOSER_URL="http://localhost:3020/"
+RABBITMQ_URL="amqp://guest:guest@localhost:5672"
+```
+
+#### nowCRM Environment Variables
+```
+NODE_ENV='development' # Options: 'development', 'production', 'test'
+
+CRM_BASE_URL="http://localhost:3000"
+
+CRM_STRAPI_API_URL="http://localhost:1337/api/"
+CRM_STRAPI_API_TOKEN=""
+DAL_URL='http://localhost:6001/api/'
+
+COMPOSER_URL="http://localhost:3020/"
+
+# API URLs
+STRAPI_URL="http://localhost:1337/api/"
+JOURNEYS_URL="http://localhost:3010/"
+
+# Authentication
+CRM_STRAPI_API_TOKEN=""
+CRM_TOTP_ENCRYPTION_KEY=""
+AUTH_SECRET="pZsHmI9P7wcs03/BEuFtMxi9HbSuyCwyknuyx7BIads="
```

Comment:
🛑 **Gitleaks** has detected a secret with rule-id `generic-secret` in commit 183b0008044b7a640d8f82c11688371bc20c868f.
If this secret is a _true_ positive, please rotate the secret ASAP.

If this secret is a _false_ positive, you can add the fingerprint below to your `.gitleaksignore` file and commit the change to this branch.

```
echo 183b0008044b7a640d8f82c11688371bc20c868f:README.md:generic-secret:625 >> .gitleaksignore
```